mobly/controllers/android_device_lib/snippet_client.py

```diff
@@ -173,7 +173,7 @@ def _start_app_and_connect(self):
     cmd = _LAUNCH_CMD.format(shell_cmd=persists_shell_cmd,
                              user=self._get_user_command_string(),
                              snippet_package=self.package)
-    start_time = time.time()
+    start_time = time.monotonic()
```

Comment:
do we want to use `perf_counter` instead since it's measuring time diff here?

Reply:
`time.perf_counter()` would change about 33,491 times more often than `time.monotonic()`, it's more accurate.